src/parser/resources/item/item_objects.py

```diff
@@ -0,0 +1,32 @@
+import sys
+import os
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '../..')))
+import resources.resource as resource
+
+
+# extends Resource
+class Item(resource.Resource):
+    objects = {}
+    chart = None
+
+    def __init__(self, key):
```

Comment:
To me it makes more sense if each resource takes in the data immediately, instead of happening outside of the class. That keeps it self contained, and allows us to have any specific validation if we want